docs/source/user-guide/integrations/h2o.rst

```diff
@@ -0,0 +1,157 @@
+============================
+H2O
+============================
+
+`H2O <https://docs.h2o.ai/h2o/latest-stable/h2o-docs/automl.html>`__ is
+a widely used AutoML framework, written in Java but with multiple fully
+implemented API libraries, including one in python. In this example,
+we’ll be showing how you can effortlessly test your H2O model with
+deepchecks in the exact same way you can test any sklearn model.
+
+This example can be run using
+`colab <https://colab.research.google.com/github/deepchecks/deepchecks/tree/examples/integrations/h2o/deepchecks_h2o_tutorial.ipynb>`__.
+
+Building an H2O Model
+---------------------
+
+First we’ll load our dataset, using the adult dataset that can be
+downloaded from the `UCI machine learning
+repository <http://archive.ics.uci.edu/ml>`__ and is also available
+in `deepchecks.tabular.datasets`.
+
+.. code:: ipython3
+
+    from deepchecks.tabular.datasets.classification import adult
+
+.. code:: ipython3
+
+    train_df, test_df = adult.load_data(data_format='Dataframe', as_train_test=True)
+
+Now we’ll build an H2O `H2OGradientBoostingEstimator <https://docs.h2o.ai/h2o/latest-stable/h2o-py/docs/modeling.html#h2o.estimators.gbm.H2OGradientBoostingEstimator>`__ based on this
+dataset split.
+
+.. code:: ipython3
+
+    import h2o
+    h2o.init()
+
+
+H2O models can only train and predict on H2OFrame objects.
+
+.. code:: ipython3
+
+    train_h2o = h2o.H2OFrame(train_df)
+
+.. code:: ipython3
+
+    gbm = h2o.estimators.gbm.H2OGradientBoostingEstimator();
+
+We must tell the H2O model what are the feature columns and what is the
+name of the label column.
+
+.. code:: ipython3
+
+    feature_cols = adult._FEATURES
+    label_col = adult._target
+
+
+.. code:: ipython3
+
+    gbm.train(x=feature_cols, y=label_col, training_frame=train_h2o, verbose=False);
+
+
+To compare to our new H2O model, we’ll download the default scikit-learn
+model (a pipeline containing a naive Random Forest Classifier) for this
+task from deepchecks.
+
+.. code:: ipython3
+
+    skmodel = adult.load_fitted_model()
+
+Running deepchecks on H2O
+=========================
+
+before we can run deepchecks on our H2O model, we have one step
+remaining - bridging between the API of the H2O model and the
+scikit-learn API that deepchecks expects.
+
+We have created a simple wrapper for our H2O model, that implements the required API for deepchecks according the the
+:doc: `</user-guide/tabular/supported_models>` guide. This wrapper is available from
+`deepchecks.tabular.integrations.h2o`.
+
+Generally the wrapper model will contain 2 functions in
+case of a classification problem: the ``predict`` and the ``predict_proba`` functions that will be called by
+deepchecks. In addition, it is also possible to specify the feature importances of the model. Read more about
+feature importance handling in the :doc: `</user-guide/tabular/feature_importance>` guide.
```

Comment:
Would you say it is possible or recommended?
(do h2o models have it built-in? if so we can demonstrate also how etc.)

Asking here since the feature importance topic is a sensitive one computation-time wise (and affect user's experience)

Reply:
In this case it isn't computed to to long projected time, so would say that in that case if you want it, you better give it yourself.